public/app/plugins/panel/geomap/components/MeasureOverlay.tsx

```diff
@@ -0,0 +1,313 @@
+import { css } from '@emotion/css';
+import Map from 'ol/Map';
+import { Geometry, LineString, Point } from 'ol/geom';
+import { Draw, Modify } from 'ol/interaction';
+import { Vector as VectorLayer } from 'ol/layer';
+import { Vector as VectorSource } from 'ol/source';
+import { getArea, getLength } from 'ol/sphere';
+import { Circle as CircleStyle, Fill, RegularShape, Stroke, Style, Text } from 'ol/style';
+import React, { useState } from 'react';
+import tinycolor from 'tinycolor2';
+
+import { GrafanaTheme } from '@grafana/data';
+import { IconButton, RadioButtonGroup, stylesFactory } from '@grafana/ui';
+import { config } from 'app/core/config';
+
+type Props = {
+  map: Map;
+  menuActiveState: (value: boolean) => void;
+};
+
```

Comment:
NIT:  order wise, it is odd to have all the utility functions/constants between the Props and the component they work with.  I like to have

```
type Props { ...} 

export TheCompoent( {xxx}: Props ) { ... }

...
other stuff you need

```

I am also a little skeptical that this creates all the style objects directly in global constants, but is only ever used very very rarely.  Is there a way that we only create the more complex static objects when needed?